src/main/java/com/synopsys/integration/jenkins/scan/service/BlackDuckParametersService.java

```diff
@@ -100,5 +106,4 @@ public Map<String, Object> createBlackDuckParametersMapFromJenkinsUI() {
 
```

Comment:
Here we noticed a bug, if the global configuration is not done then `config.getBlackDuckUrl()` and `config.getBlackDuckCredentialsId()` produce null pointer exception. So we need to handle this. We may return the empty map if these values are empty or null.